src/components/postgresConnect.ts

```diff
@@ -656,8 +656,15 @@ class PostgresDB {
     }
   }
 
-  public isUnique = async (table: string, field: string, value: string): Promise<boolean> => {
-    const text = `SELECT COUNT(*) FROM "${table}" WHERE LOWER(${field}) = LOWER($1)`
+  public isUnique = async (
+    table: string,
+    field: string,
+    value: string,
+    caseSensitive: boolean
+  ): Promise<boolean> => {
+    const text = caseSensitive
+      ? `SELECT COUNT(*) FROM "${table}" WHERE(${field}) =($1)`
+      : `SELECT COUNT(*) FROM "${table}" WHERE LOWER(${field}) = LOWER($1)`
```

Comment:
```suggestion
    const text = caseSensitive
      ? `SELECT COUNT(*) FROM "${table}" WHERE LOWER(${field}) = LOWER($1)`
      : `SELECT COUNT(*) FROM "${table}" WHERE ${field} = $1`
```
Your one's round the wrong way -- we want "caseSensitive" (i.e. the default) to use the existing query -- only when "caseSensitve = false" should it use the non-casesensitive query.

You also don't need the `( )` round the values in the new one -- they were just for the `LOWER()` function.

Reply:
I see it now,

The double negatives were playing tricks on my brain.

Reply:
I'm thinking now it should be the other way.

Typical calling for unique is not caseSensitive. So false should be standard. This particular call is true caseSensitive. So I want to change both files to the reverse.

What do you think?

Reply:
Yeah, I see what you mean. Default should be NOT case-sensitive. Cool, as long as we don't have to change existing implementations other than the reset-password form.

Other option is to just rename the parameter to "caseInsensitive" and leave the default as `true`. Up to you.